fe/fe-core/src/main/java/com/starrocks/plugin/AuditEvent.java

```diff
@@ -55,8 +55,13 @@ public enum EventType {
     public long timestamp = -1;
     @AuditField(value = "Client")
     public String clientIp = "";
+    // The original login user
     @AuditField(value = "User")
     public String user = "";
+    // The user used to authorize
```

Comment:
If userA->userB->userC, the impersonate list is a long list. If the userC executes the query, the print userB or userA?